python/docs/conf.py

```diff
@@ -73,21 +73,44 @@
     "cmdstanpy": ("https://mc-stan.org/cmdstanpy/", None),
 }
 
-# julia doc build
+
+breathe_projects = {"bridgestan": "./_build/cppxml/"}
+breathe_projects_source = {"bridgestan": ("../../src/", ["bridgestan.h", "bridgestanR.h"])}
+breathe_default_project = "bridgestan"
+breathe_doxygen_config_options = {"EXCLUDE_SYMBOLS": "to_csv"}
+
+
+# Julia and C++ doc build
 import os
 import subprocess
 import pathlib
 
+RUNNING_IN_CI = os.environ.get("CI") or os.environ.get("READTHEDOCS")
+
 try:
     print("Building Julia doc")
-    subprocess.run(
-        ["julia", '--project=.', "./make.jl"],
-        cwd=pathlib.Path(__file__).parent.parent.parent / "julia" / "docs",
-        check=True,
-    )
+    # subprocess.run(
```

Comment:
Will you remind me why this is commented out?

Reply:
My mistake when trying to iterate quicker on the docs